test/unit/test-amp-pixel.js

```diff
@@ -99,20 +99,23 @@ describes.realWin('amp-pixel', {amp: true}, env => {
   });
 
   it('should disallow http URLs', () => {
+    expectAsyncConsoleError(/src attribute must start with/);
```

Comment:
Nit, can reuse the `rejectedWith` content by creating a single const reference.

Reply:
Thanks for the review. This change has been made.